content/en/seachat/full-tutorials/01-mailerlite.md

```diff
@@ -0,0 +1,71 @@
+
+The second field, "Group ID" is optional. If you want users to be added to specific groups in your mailing list you can add their ids here (separated by comma if multiple). 
+
+Once you have completed configuring your forms, you can save your configurations by pressing the save button at the bottom of the page. 
+
+If you wish to remove your configurations, you can use the remove button at the top right of the page. This will clear out your settings for the mailerlite form in your workspace, filling the page again with the default values. 
+
+## Step 3: Test Your Form
+
+<img width="100%" style="border-radius: 0.4rem" src="/images/seachat-integrations/mailerlite/navigate-to-agent-info.png" alt="mailerlite integration setup page.">
+Once you have configured and saved your Mailerlite form settings, you probably want to test that everything is working. To do this, make sure your form is enabled and navigate to the "Agent Information" page from the Agent Configuration drop down on the left action bar.
+
+<img width="100%" style="border-radius: 0.4rem" src="/images/seachat-integrations/mailerlite/test-ai-agent-button.png" alt="mailerlite integration setup page.">
+On this page you will see a large green button that says "Test AI Agent" on the bottom right. If you click this button it will open a chat conversation with your currently selected AI Agent. 
+
+<img width="100%" style="border-radius: 0.4rem" src="/images/seachat-integrations/mailerlite/mailerlite-form-submission.png" alt="mailerlite integration setup page.">
+If it is a new conversation, within a few seconds your Mailerlite form that you configured in the last step will appear at the bottom of the chat window. From here you can fill out requested information and hit submit at the bottom of the form. (Note: If this is not your first time testing your AI Agent it is possible that it will open a previous conversation. You can open this page in a new incognito window to ensure you start a new conversation)  
+
+<img width="100%" style="border-radius: 0.4rem" src="/images/seachat-integrations/mailerlite/mailerlite-find-email.png" alt="mailerlite integration setup page.">
+Once the form has been submitted you can head over to mailerlite. Once you log in you can go to the Subscribers page from the left action bar. Here, you should be able to see the email from the user that you have just submitted at the top of your subscriber list. If you dont see it right away, you can also search for it up with the grey search button at the top right of the page.
+
+<img width="100%" style="border-radius: 0.4rem" src="/images/seachat-integrations/mailerlite/mailerlite-view-email.png" alt="mailerlite integration setup page.">
+You can click on the email to view details about the newly added subscriber such as any other information that might have been submitted with the form, as well as any groups that you included on the setup page.
```

Comment:
After this final paragraph, add a paragraph explaining why SeaChat can bring a smooth pipeline of new leads and how one can stay in touch of the subscribers on MailerLite